ods-tutorial/source/dashboard_modules/how_to_create_tabs.rst

```diff
@@ -0,0 +1,546 @@
+
+
+.. rst-class:: block-step
+
+    18 + Our ``ng-class`` directive works! But it isn't so pretty, is it? Let's go back to the CSS section to add the finishing touches! Here, we're going to add more space around the tab titles to avoid the border to fall into the tab content area. We are also going to highlight the possible selection of a tab title by building a specific style only visible when the mouse hovers the tab titles.
+
+.. code-block:: css
+
+ .tabs {
+    margin: 1em; /* Adds lots of space around the tabs, to avoid the border to fall into the tab content area */
+ }
+
+ /* Only visible when the mouse is on a tab title */
+ .tab:hover {
+    color: black; /* Colors the text in black to highlight the possible selection */
+    text-decoration: none; /* Removes the hyperlink underline style */
+ }
+
+.. rst-class:: img-hide
+
+    SCREENSHOT
+
+.. rst-class:: block-step
+
+    19 + Go back to the HTML section to apply the whole new CSS style. Our ``tabs`` CSS class will do very well in the ``div`` tag that wraps all the tab titles. As for ``tab:hover``, since it is built on top of the main CSS class ``tab``, it automatically applies to the HTML elements with the ``tab`` CSS class.
+
+.. code-block:: html
+
+ <div class="container">
+   <div class="ods-box" ng-init="tabselector = 'firsttab'">
+
+     <div class="tabs">
+       <a class="tab"
+          ng-click="tabselector = 'firsttab'"
+          ng-class="{'activetab' : tabselector == 'firsttab'}">1st tab title</a>
+       <a class="tab"
+          ng-click="tabselector = 'secondtab'"
+          ng-class="{'activetab' : tabselector == 'secondtab'}">2nd tab title</a>
+       <a class="tab"
+          ng-click="tabselector = 'thirdtab'"
+          ng-class="{'activetab' : tabselector == 'thirdtab'}">3rd tab title</a>
+     </div>
+
+     <div class="ods-box">
+       <div ng-if="tabselector == 'firsttab'">
+         Content of first tab
+       </div>
+       <div ng-if="tabselector == 'secondtab'">
+         Content of second tab
+       </div>
+       <div ng-if="tabselector == 'thirdtab'">
+         Content of third tab
+       </div>
+     </div>
+
+   </div>
+ </div>
+
+.. rst-class:: img-hide
+
+    SCREENSHOT
+
+.. rst-class:: block-step
+
+    20 + There we go! We have created our very own tabs! The only thing left to do is to replace every example text by real content.
+
+You are free to choose the tab titles and to add your own tab contents. For this tutorial, will complete the tabs using the `Shanghai World University Ranking <https://data.opendatasoft.com/explore/dataset/shanghai-world-university-ranking%40public-us/>`_ dataset. The first tab will contain the table visualization, the second tab will contain a map created with the Map Builder and the third tab will contain a chart visualization. Each tab title will of course indicate what the tab content is about, and they will be preceded by a `Font Awesome <https://fontawesome.com/>`_ icon (just to spice things up a bit!).
```

Comment:
The link is not valid anymore, the final and public version is now released here 
https://data.opendatasoft.com/explore/dataset/shanghai-world-university-ranking%40public
![](https://media.giphy.com/media/dWm5HKuKjZTO/giphy.gif)